internal/service/file/file_folder.go

```
@@ -308,6 +309,13 @@ func (s *FileService) getUniqueFilename(ctx context.Context, name, parentID, ten
 
 // CreateFolder creates a new folder or virtual file
 func (s *FileService) CreateFolder(ctx context.Context, tenantID, name, parentID, fileType string) (map[string]interface{}, error) {
+	// "/" is the root folder name and the path separator used by recursive
+	// folder creation, so names containing it collide with the root folder
+	// and break path-based lookups.
+	if strings.Contains(name, "/") {
+		return nil, errors.New(`Folder name cannot contain "/"`)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)